CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# How to Contribute
+
+## What is this document?
+
+This document outlines a generic process of contributing and applies to all CHAOSS repositories. Each repository may have unique guidelines specific to the project.
```

Comment:
So it's a good thing to have a CONTRIBUTING.md file at the root of every repo for the reasons outlined on [GitHub's annoucement post](https://github.com/blog/1184-contributing-guidelines).

Reply:
Agreed. My thought was to leave repository specific elements in those files and move common elements in this CONTRIBUTING.md and link to it.

Reply:
> We don't need to explain how to use git and forks on every repository.

You could even link to some external documentation for that, fwiw.

Reply:
Linking to resources is what I tried to do below. Suggestions for additional resources?